easydigitaldownloads/easy-digital-downloads
# Change event to 'insert' #8427

Fixes #8427

Proposed Changes:
1. Change the event from `select` to `insert`.

To test:

- Add/edit a download
- Add a file to that download
- Ensure the file URL appears in the meta box and that after saving the download, the file is still attached.
